### PR TITLE
RS-1255 FE-T Webpack SourceMap Fix For JSX Files

### PR DESCRIPTION
**Pull request by ianwcarlson**
* Changed regex expression to include .js or .jsx files

RS-1255

### CODE REVIEW

File: lib/SourceMapDevToolPlugin.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ SourceMapDevToolPlugin.prototype.apply = function(compiler) {
 	var fallbackModuleFilenameTemplate = this.fallbackModuleFilenameTemplate;
 	var requestShortener = new RequestShortener(compiler.context);
 	var options = this.options;
-	options.test = options.test || /\.(js|css)($|\?)/i;
+	options.test = options.test || /\.(js(x)|css)($|\?)/i;
```
Comment (johnharger) [inline lib/SourceMapDevToolPlugin.js]: is this correct? Looks like it will skip js files.. needs to make x optional? I think

Comment (ianwcarlson) [inline lib/SourceMapDevToolPlugin.js]: You're right.  I think I fixed.  I tested on a regex tester.

**Comment by johnharger** [comment]
@ianwcarlson Looks good.

@cromanozero Ready to merge.

**Comment by ianwcarlson** [comment]
@cromanozero bump

**Pull request merged**